packages/core/content-manager/server/src/history/services/history.ts

```diff
@@ -112,7 +112,7 @@ const createHistoryService = ({ strapi }: { strapi: LoadedStrapi }) => {
           where: {
             $and: [
               { contentType: params.contentType },
-              { relatedDocumentId: params.documentId },
+              ...(params.documentId ? [{ relatedDocumentId: params.documentId }] : []),
```

Comment:
I think they are still there, here we are jest fetching, but the single types should still be created with a relatedDocumentId. Unless its a publish action since I believe there is a bug there (see my other PR for e2e)

Reply:
ah i got confused, this is the findPage not the creation stuff